syntax = "proto3";

package your_package_name;

service HelloService {
    rpc SayHello (HelloRequest) returns (HelloResponse) {}
}

message HelloRequest {
    // Add message fields here
}

message HelloResponse {
    string message = 1;
    int64 timestamp = 2;
}
